book.bib

```diff
@@ -486,4 +486,79 @@ @article{vandewalle2009reproducible
   year={2009},
   publisher={IEEE}
 }
-
+@article{Goodman2016,
+  doi = {10.1126/scitranslmed.aaf5027},
+  url = {https://doi.org/10.1126/scitranslmed.aaf5027},
+  year = {2016},
+  month = jun,
+  publisher = {American Association for the Advancement of Science ({AAAS})},
+  volume = {8},
+  number = {341},
+  pages = {341ps12--341ps12},
+  author = {Steven N. Goodman and Daniele Fanelli and John P. A. Ioannidis},
+  title = {What does research reproducibility mean?},
+  journal = {Science Translational Medicine}
+}
+@article{Marwick2016,
+  doi = {10.1007/s10816-015-9272-9},
+  url = {https://doi.org/10.1007/s10816-015-9272-9},
+  year = {2016},
+  month = jan,
+  publisher = {Springer Science and Business Media {LLC}},
+  volume = {24},
+  number = {2},
+  pages = {424--450},
+  author = {Ben Marwick},
+  title = {Computational Reproducibility in Archaeological Research: Basic Principles and a Case Study of Their Implementation},
+  journal = {Journal of Archaeological Method and Theory}
+}
+@article{Piccolo2016,
+  doi = {10.1186/s13742-016-0135-4},
+  url = {https://doi.org/10.1186/s13742-016-0135-4},
+  year = {2016},
+  month = jul,
+  publisher = {Oxford University Press ({OUP})},
+  volume = {5},
+  number = {1},
+  author = {Stephen R. Piccolo and Michael B. Frampton},
+  title = {Tools and techniques for computational reproducibility},
+  journal = {{GigaScience}}
+}
+@article{Piccolo2016,
+  doi = {10.1186/s13742-016-0135-4},
+  url = {https://doi.org/10.1186/s13742-016-0135-4},
+  year = {2016},
+  month = jul,
+  publisher = {Oxford University Press ({OUP})},
+  volume = {5},
+  number = {1},
+  author = {Stephen R. Piccolo and Michael B. Frampton},
+  title = {Tools and techniques for computational reproducibility},
+  journal = {{GigaScience}}
+}
+@article{Kluyver ThomasRagan-Kelley BenjaminP&eacute;rez FernandoGranger BrianBussonnier MatthiasFrederic JonathanKelley KyleHamrick JessicaGrout JasonCorlay Sylvainet al.2016,
```

Comment:
Bibliography don not work due to the corrupted BibTeX entry.

This line is please remove unnecessary part "ThomasRagan-Kelley BenjaminP&eacute;rez FernandoGranger BrianBussonnier MatthiasFrederic JonathanKelley KyleHamrick JessicaGrout JasonCorlay Sylvainet al.2016,"

```@article{Kluyver ThomasRagan-Kelley BenjaminP&eacute;rez FernandoGranger BrianBussonnier MatthiasFrederic JonathanKelley KyleHamrick JessicaGrout JasonCorlay Sylvainet al.2016,
```